apisix/plugins/grpc-web.lua

```diff
@@ -60,6 +59,9 @@ function _M.check_schema(conf)
 end
 
 function _M.access(conf, ctx)
+    -- set context variable mime
+    ctx.grpc_web_mime = core.request.header(ctx, "Content-Type")
```

Comment:
better to add a comment explaining why you need to set the ctx.grpc_web_mime in advance

Reply:
added